api/src/main/java/org/openmrs/module/commonreports/reports/DRCARTSecondLineReportManager.java

```diff
@@ -0,0 +1,252 @@
+		sqd.setQuery(sql);
+		sqd.addParameter(new Parameter("onOrAfter", "On Or After", Date.class));
+		sqd.addParameter(new Parameter("onOrBefore", "On Or Before", Date.class));
+		
+		// Alive patients
+		BirthAndDeathCohortDefinition livePatients = new BirthAndDeathCohortDefinition();
+		livePatients.setDied(false);
+		
+		CompositionCohortDefinition ccd = new CompositionCohortDefinition();
+		ccd.initializeFromElements(sqd, livePatients);
+		
+		SqlCohortDefinition sqd2 = new SqlCohortDefinition();
+		
+		// Visit in range
+		// ART regimen -> AZT/3TC +ATV/r
+		String sql2 = "SELECT DISTINCT p.patient_id FROM patient p WHERE p.voided = 0 "
+		        + "AND EXISTS (SELECT 1 FROM visit v WHERE v.patient_id = p.patient_id AND v.date_started BETWEEN :onOrAfter AND :onOrBefore AND v.voided = 0) "
+		        + "AND EXISTS (SELECT 1 FROM obs o JOIN concept c_question ON o.concept_id = c_question.concept_id JOIN concept c_answer ON o.value_coded = c_answer.concept_id WHERE o.person_id = p.patient_id AND c_question.uuid = '164432AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA' AND c_answer.uuid = '62243dfe-6284-48a9-afcc-63149ac6d709' AND o.voided = 0   AND o.obs_datetime = (SELECT MAX(do_inner.obs_datetime) FROM obs do_inner WHERE do_inner.person_id = p.patient_id AND do_inner.concept_id = c_question.concept_id AND do_inner.voided = 0) );";
+		sqd2.setQuery(sql2);
+		sqd2.addParameter(new Parameter("onOrAfter", "On Or After", Date.class));
+		sqd2.addParameter(new Parameter("onOrBefore", "On Or Before", Date.class));
+		
+		CompositionCohortDefinition ccd2 = new CompositionCohortDefinition();
+		ccd2.initializeFromElements(sqd2, livePatients);
+		
+		artSecondLine.addRow(MessageUtil.translate("commonreports.report.drc.AZT+3TC+LPV+r"), ccd, parameterMappings);
+		artSecondLine.addRow(MessageUtil.translate("commonreports.report.drc.AZT+3TC+ATV+r"), ccd2, parameterMappings);
+		
+		setColumnNames();
+		
+		GenderCohortDefinition males = new GenderCohortDefinition();
+		males.setMaleIncluded(true);
+		artSecondLine.addColumn(col1, createCohortComposition(males), null);
+		
+		GenderCohortDefinition females = new GenderCohortDefinition();
+		females.setFemaleIncluded(true);
+		artSecondLine.addColumn(col2, createCohortComposition(females), null);
+		
+		GenderCohortDefinition allGenders = new GenderCohortDefinition();
+		allGenders.setFemaleIncluded(true);
+		allGenders.setMaleIncluded(true);
+		artSecondLine.addColumn(col3, createCohortComposition(allGenders), null);
+		
+		// 10-14 years
+		AgeCohortDefinition _10To14y = new AgeCohortDefinition();
```

Comment:
If we're not distinguishing by age, should we have the other age disaggregations here or does that not make clinical sense?